packages/nextlove-generators/tsconfig.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "../nextlove/tsconfig.json",
```

Comment:
🐧 I'd probably hoist this to a common location

Reply:
let's do that in another PR